.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,50 @@
-<!--- Prepend PR title with [WIP] if work in progress. Remove when ready for review. -->
-
-<!--- For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments -->
+<!--
+  For Work In Progress Pull Requests, please use the Draft PR feature,
+  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
+  
+  Before submitting a Pull Request, please ensure you've done the following:
+  - 📖 Read the Contributing Guide: https://github.com/codebuddies/frontend/blob/main/CONTRIBUTING.md.
+  - 📖 Read the Open Sauced Code of Conduct: https://github.com/codebuddies/frontend/blob/main/CODE_OF_CONDUCT.MD.
+  - 👷‍♀️ Aim to create small PRs by splitting up feature work, and potentially multiple issues. 
+  - ✅ Provide tests for your changes if relevant.
+  - 📝 Use descriptive commit messages.
+  - 📗 Update any related documentation and include any relevant screenshots.
+  Note: h/t to https://github.com/open-sauced/open-sauced/blob/main/.github/PULL_REQUEST_TEMPLATE.md for this pull request template model. 
+-->
 
 ## What type of PR is this? (check all applicable)
 
-- [ ] Refactor
-- [ ] Feature
-- [ ] Bug Fix
-- [ ] Documentation Update
+- [ ] ♻️ Refactor
+- [ ] ✨ Feature
+- [ ] 🐛 Bug Fix
+- [ ] 🎨 Enhancement
+- [ ] 📝 Documentation Update
+- [ ] 🔖 Release
+- [ ] 🚩 Other
+
+## Context
+
```

Comment:
@lpatmo it'd be great to have some place holder examples for these sections.

Reply:
Good idea! Added some examples in e46bdfe